guides/source/routing.md

```diff
@@ -1028,7 +1069,7 @@ This will generate the singular URL `/basket` instead of the usual `/baskets/:id
 Customizing Resourceful Routes
```

Comment:
After reading about "Resourceful Routes" the customization might go unnoticed if the next section is called "Non-Resourceful Routes".

Would it make sense to move this whole section under "Resource Routing: the Rails Default"?
In this section we mention constraints though, and we currently explain constraints in "Non-Resourceful Routes".

Or should we add a last section to "Resourceful Routes" that links to this section?

Reply:
Sure, added a link. As the "customizing" one is a top level section, it'll be visible in the right navigation and such that should help too.